sql/core/src/main/scala/org/apache/spark/sql/internal/BaseSessionStateBuilder.scala

```diff
@@ -81,8 +81,8 @@ abstract class BaseSessionStateBuilder(
   /**
    * SQL-specific key-value configurations.
    *
-   * These either get cloned from a pre-existing instance or newly created. The conf is always
-   * merged with its [[SparkConf]].
```

Comment:
Hi, @viirya . Is this the only one left?

Reply:
I think so. I don't see other left.